BUILDING.md

```diff
@@ -1,40 +1,59 @@
 ### Building
 
-To build coremltools from source, you require you need
-[CMake](https://cmake.org) and
-[Miniconda](https://docs.conda.io/en/latest/miniconda.html) to configure the
-project.
+This page describes building Core ML Tools (coremltools) from the source repository.
 
-Our makefile & scripts require the **zsh** shell (default shell for macOS
-10.16+) installed in `/usr/bin`.
+## Requirements
 
-The following targets will handle the development environment for you. If you
-need to add packages, edit the reqs/pip files and the auto-environment will
-install them automatically.
+To build coremltools from source, you need the following:
 
+* [CMake](https://cmake.org/)
+* [Miniconda](https://docs.conda.io/en/latest/miniconda.html) (or [Anaconda](https://www.anaconda.com/))
+* [Zsh shell](http://zsh.sourceforge.net/) (the default shell for macOS 10.16+) installed in /usr/bin
 
-* `build` | Build coremltools in *debug* mode (include symbols).
+## Build script
+
+Follow these steps:
+
+1. Fork and clone the GitHub [coremltools repository](https://github.com/apple/coremltools).
+2. Run the [build.sh](scripts/build.sh) script to activate the appropriate release environment and build `coremltools`. The build.sh script uses Python 3.7, but you can include `--python=3.5` (or `3.6`, `3.8`, and so on) as a argument to change the Python version. The script creates a new `build` folder with the coremltools distribution, and a `dist` folder with Python wheel files.
```

Comment:
I'm not sure "appropriate release environment" really makes sense. I don't think I've heard anyone say "release environment" before.

`build.sh` actually creates the environment if it's the first time you're running it. Otherwise it just uses it.

I would say "By default this script uses Python 3.7" rather than "The build.sh script uses Python 3.7"

Reply:
OK, done. "Run the [build.sh](scripts/build.sh) script to create and activate the environment (or use it if it already exists), and to build `coremltools`. By default this script uses Python 3.7..."